### ISSUE TITLE
v29.2 - Focus cell and press any key resets row height

### ISSUE THREAD

**Issue by Paveltrof86** [ag-grid/ag-grid#6410]
Hello!
We`ve updated **Ag Grid to v.29.2** and noticed some strange behaviour.
**Files:
ag-grid-enterprise.min.noStyle.js
ag-theme-balham.css
ag-grid.css**

If I focus on any cell and then I press any key -> all rows set the same height. **(27 px)**

![line height](https://user-images.githubusercontent.com/19744837/228910130-e1337ec7-60b3-441b-a6ef-73899ca38ac4.jpg)

https://user-images.githubusercontent.com/19744837/228891917-3b2fea09-5745-4a59-95cc-9b9b9f93e76f.mp4

Everything works fine with v29.0.0

**Comment by Paveltrof86** [comment]
For every column I use **autoHeight: true** now.
If I disable **autoHeight: true** and set e.g. getRowHeight: 100 - works fine.

I`ve added **suppressCellFocus: true** to grid options and now the row heights the same after press any key.

**Comment by StephenCooper** [comment]
Can you share the reproduction that you used there?

**Comment by Paveltrof86** [comment]
```
gridOptions = {
     rowData: rowData,
     columnDefs: columnDefs,
     defaultColDef: {
            autoHeight: true,
     }
     onModelUpdated: (params) => {
            console.log("onModelUpdated");
     },
}
```
Everything else I have commented


ColDef example. I`ve removed cellRenderer:

```
{
field : `NAME0KSTable298Sum`,
headerName : `ABCDTEst (3)`,
comparator : ,
valueGetter : params => params.node.group ? null : params.data...,
filter : 'agMultiColumnFilter',
filterParams : ,
chartDataType : 'category',
headerComponentParams : ,
cellStyle : {textAlign: ''}
},
{
field : `CLASS1KSTable298Sum`,
headerName : `<span class="icon-message" title="Comments"></span>`,
comparator : ,
valueGetter : params => params.node.group ? null : params.data...,
filter : 'agMultiColumnFilter',
filterParams : ,
chartDataType : 'category',
headerComponentParams : {,
cellStyle : {textAlign: 'center'}
},
{
field : `DESCRIPTION2KSTable298Sum`,
headerName : `Description`,
comparator : ,
valueGetter : params => params.node.group ? null : params.data...,
filter : 'agMultiColumnFilter',
filterParams : ,
chartDataType : 'category',
headerComponentParams : ,
cellStyle : {textAlign: ''}
}

```

**Comment by StephenCooper** [comment]
I can see the issue in your videos but I am struggling to reproduce it myself. 

To investigate this further I will need a code reproduction that I can run myself.

**Comment by Paveltrof86** [comment]
Fix in v29.3.0
Thank you!

**Comment by StephenCooper** [comment]
Thanks for letting us know it's fixed!
